template.yaml

```diff
@@ -2799,7 +2799,7 @@ Resources:
               NoncurrentDays: !Ref S3NoncurrentVersionExpirationDays
       Tags:
           - Key: IncludedInBackup
-            Value: "false"
+            Value: "true"
```

Comment:
Do we need both? Zip bucket contains files from xml bucket in zip file. I would ask JEG if we simply want to keep all zip files in zip bucket + include in backup. (remove DeleteContentAfter3Days policy on ScopusZipBucket)